src/components/NewSnippet.js

```diff
@@ -48,7 +55,29 @@ const Input = styled.input`
   width: 470px;
 `;
 
+const Message = styled.p`
+  font-size: 13px;
+  margin-top: 5px;
+  color: #EE0004;
+  text-align: right;
+`;
+
 export default function NewSnippet() {
+  const [failureReason, setFailureReason] = useState("");
+
+  const hashtagInput = useRef();
+
+  const handleButtonClick = () => {
+    const hashtags = hashtagInput.current.value;
+    const hashtagList = hashtags.split(" ");
```

Comment:
hashtags와 hashtagList는 동일한 뜻을 가져서 변수명만으로 차이점을 알아차리기는 어려울 것 같습니다!
아래와 같이 붙여 쓰거나, 혹은 split의 결과라는 것을 직접적으로 나타내주는 것도 좋을 것 같습니다 👍 

```suggestion
    const hashtags = hashtagInput.current.value.split(" ");
    // or
    const hashtags = hashtagInput.current.value;
    const splitedHashtags = hashtags.split(" ");
```